recipes/coursera-dl/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "coursera-dl" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "109100995aac2e174502ca043ce9d595092d20bd5be4401faa1a85c0516ca4ba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - coursera-dl=coursera.coursera_dl:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - beautifulsoup4 >=4.1.3
+    - requests >=2.10.0
+    - six >=1.5.0
+    - urllib3 >=1.10
+    - pyasn1 >=0.1.7
+    - keyring >=4.0
+
+test:
+  imports:
+    - coursera
```

Comment:
Could we please test the entrypoint like so? A different option can be used if needed.

```yaml
  commands:
    coursera-dl --help
```